### DISCUSSION TITLE
Fullscreen and Remove Close Button

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/microsoft-ui-xaml#9593] [Q&A] [answered]
Using the latest WindowsAppSDK and trying to convert a UWP app which is a full screen app. I'm trying this line out on a WinUI3 app made with Template Studio and it doesn't work. I can't figure out how to remove the X (close) button either.

```
    public MainWindow()
    {
        InitializeComponent();
        AppWindow.SetPresenter(AppWindowPresenterKind.FullScreen);
        ExtendsContentIntoTitleBar = true;
    }

**Comment by DarranRowe** [comment]
Do you need to use ExtendsContentIntoTitleBar?
As the documentation for [FullScreenPresenter](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.windowing.fullscreenpresenter?view=windows-app-sdk-1.5) states, 

"Full-screen mode is not the same as maximized. When an app is in full-screen mode, it takes up the entirety of the screen. System elements, like **title bars**, status bars, or the taskbar, are hidden by default."

The bold emphasis is mine. So, there is an issue here where ExtendContentIntoTitleBar should not activate when there is no title bar style, but otherwise, what do you believe should happen when you try to extend content into something that doesn't exist?

This is what I get when I only set the presenter:

![Screenshot 2024-04-29 164422](https://github.com/microsoft/microsoft-ui-xaml/assets/52577874/f6c22925-4960-4e34-abf6-eb4b6ee95ff8)

**Reply by myokeeh** [reply, marked as answer]
I was initially trying to extend to the titlebar and hide the minimize, maximize, close buttons to see the effect I was going for in a Windowed state. I later added `AppWindow.SetPresenter(AppWindowPresenterKind.FullScreen);`. Regardless, it's working now, but I don't understand why it wasn't working yesterday.
